src/Domain/SalesforceProxyRepository.php

```diff
@@ -67,9 +49,4 @@ protected function getClient(): Client\Common
 
         return $this->client;
     }
-
-    protected function getLogger(): ?LoggerInterface
```

Comment:
property was typed as non-nullable so no reason to make return it as nullable here. And then a protected getter for a protected property is redundant.